pkg/controllers/management/node/controller.go

```diff
@@ -329,16 +330,34 @@ func aliasToPath(driver string, config map[string]interface{}, ns string) error
 				hasher.Reset()
 				hasher.Write([]byte(fileContents))
 				sha := base32.StdEncoding.WithPadding(-1).EncodeToString(hasher.Sum(nil))[:10]
-				fullPath := path.Join(baseDir, sha)
-				err := ioutil.WriteFile(fullPath, []byte(fileContents), 0600)
+				fileName := driverField
+				if ok := nodedriver.SSHKeyFields[schemaField]; ok {
+					fileName = "id_rsa"
+				}
+
+				fileDir := path.Join(baseDir, sha)
+
+				// Cleanup the old key file if it exists
```

Comment:
Can we change _"Cleanup the old key file..."_ to _"Cleanup existing key file in deprecated format..."_. The former makes it sound like it was created on last run, but really it's tech debt code that can be removed in the future.